core/src/Helpers/CoreHelpers.cs

```diff
@@ -30,11 +30,10 @@
 using System.Globalization;
 using System.Linq;
 using System.Text;
-using Microsoft.Identity.Core;
 
-namespace Microsoft.Identity.Client.Internal
+namespace Microsoft.Identity.Core.Helpers
 {
-    internal static class MsalHelpers
+    internal static class CoreHelpers
```

Comment:
>CoreHelpers [](start = 26, length = 11)

suggestion: sort these extensions into files/classes names that makes then easily discoverable?

Reply:
renamed the class to ExtensionMethods

---
In reply to: [161029979](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161029979) [](ancestors = 161029979)